src/processing_provider/provider.py

```diff
@@ -75,3 +83,19 @@ def loadAlgorithms(self):
         self.algs = self.getAlgs()
         for a in self.algs:
             self.addAlgorithm(a)
+    
+    def load(self):
+        ProcessingConfig.settingIcons[self.name()] = self.icon()
+        ProcessingConfig.addSetting(Setting(self.name(),
+                                            'SWMM_PATH',
+                                            self.tr("SWMM executable"),
+                                            'C:\\Program Files (x86)\\EPA SWMM 5.1.013\\swmm55.exe',
```

Comment:
```suggestion
                                            r'C:\Program Files (x86)\EPA SWMM 5.1.013\swmm55.exe',
```

This might be a bit more readable